design.md

```diff
@@ -1,13 +1,41 @@
 # Design
 
+asciibot
+
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
 
+Not one. We hope no one ever uses this. This is a example of what not to do.
+
+Potentially a very visual person would want it. It can be nice to read,
+although it sucks to write.
+
 ## Why did you choose this design, i.e., why did you think it would be a good idea for users to express the maze-searching computation using this syntax?
 
+We chose this design because we thought it was goofy and would be fun to parse.
+We've already done a bunch of parser combinator stuff in PLs so we thought it
+would be nice to do something else.
```

Comment:
That would explain why you chose not to use a pre-implemented parser in your parsing, which every other group did.